src/MongoDB.EntityFrameworkCore/Infrastructure/QueryableEncryptionSchemaGenerator.cs

```diff
@@ -127,8 +127,8 @@ private static BsonDocument GenerateSchemaField(IReadOnlyProperty property, stri
         var dataKeyId = property.GetEncryptionDataKeyId();
         var fieldSchema = new BsonDocument
         {
-            { "path", prefix + property.GetElementName() },
             { "keyId", dataKeyId != null ? new BsonBinaryData(dataKeyId.Value, GuidRepresentation.Standard) : BsonNull.Value },
+            { "path", prefix + property.GetElementName() },
```

Comment:
Moved path to be after keyId simply because that's the order the server returns no matter what order you send it in.